enterprise_access/apps/api/v1/tests/test_subsidy_access_policy_views.py

```diff
@@ -1091,10 +1091,16 @@ def setup_mocks(self):
         self.lms_client_instance = lms_client.return_value
         self.lms_client_instance.get_enterprise_user.return_value = TEST_USER_RECORD
 
+        includes_user_patcher = patch.object(
+            SubsidyAccessPolicy, 'includes_user'
+        )
+        self.mock_includes_user = includes_user_patcher.start()
+
         self.addCleanup(lms_client_patcher.stop)
         self.addCleanup(subsidy_client_patcher.stop)
         self.addCleanup(contains_key_patcher.stop)
         self.addCleanup(get_content_metadata_patcher.stop)
+        self.addCleanup(includes_user_patcher.stop)
```

Comment:
Seems odd to me that we'd be patching this `includes_user` function which is supposed to always return a boolean, but then not setting up a mock return value. To me, it's arbitrary what the return value of a mock function call would be before assigning `.return_value`....is it truthy or falsey?  It seems nondeterministic that it should be one or the other, and makes our unit test sensitive to upstream changes to the unit test framework libraries.

My gut is that we go ahead and toss this in:

```python
        self.mock_includes_user = includes_user_patcher.start()
        self.mock_includes_user.return_value = True
```